### DISCUSSION TITLE
Colors for engraving and cutting power?

### DISCUSSION THREAD

**Discussion by FHEMUser** [arkypita/LaserGRBL#2212] [Ideas]
The new generator for testing shows dark and light red colors for engraving power.
Does LaserGRBL now can differentiate colors ?
So whats about a color table for engraving, cutting power? 
Than we can set colors in Inkscape and import a image and can set different powers.

**Comment by LCF134** [comment]
This would be great to have. Is there a plan to implement this?

**Comment by ReggiePye** [comment]
There is a plugin available for just that called Export for LaserGRBL that saves file for cutting directly from Inkscape to LaserGRBL

https://inkscape.org/~Nikki.Smith/%E2%98%85export-to-lasergrbl-for-laser-cutting
